submissions/madmaxWMFU/Friends-App/js/app.js

```diff
@@ -0,0 +1,100 @@
+const allFriends = {
+    currentList: [],
+    changeList: [],
+    countFriends: 52,
```

Comment:
Count is a verb, so it fits for function name. Use something `numberOfFriends`, as an example